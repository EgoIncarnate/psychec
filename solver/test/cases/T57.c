// Similar to T22, but with missing declaration of node,
// that's why it's separate (triggers different solver aspects)

List3 new_node2(List3 anchor3) {
  //  no declaration of node 3
  node3->next3 = anchor3;
  return node3;
}
