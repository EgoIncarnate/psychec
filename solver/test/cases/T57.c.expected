typedef  struct __List3__ {struct __List3__ * next3;}* List3 ;
struct __List3__ * node3 ;
